lib/plugins/random-messages/index.js

```diff
@@ -0,0 +1,55 @@
+const { PLUGIN_NAME } = require("../symbols");
+
+const answers = [
```

Comment:
Lehet ez,
```suggestion
const messageReplies = [
```
 vagy ez hogy minél beszédesebb legyen
```suggestion
const knownMessageReplies = [
```